tts/structs.go

```diff
@@ -56,7 +56,7 @@ const (
 	// DeckCustomObject represents a custom card deck.
 	DeckCustomObject ObjectType = "DeckCustom"
 	// CardObject represents a card.
-	CardObject ObjectType = "Card"
+	CardObject ObjectType = "CardCustom"
```

Comment:
Could you create a new type instead, like so:

```suggestion
	CardObject ObjectType = "Card"
	// CardCustomObject represents a custom card.
	CardCustomObject ObjectType = "CardCustom"
```

And then update update this line to use `CardCustomObject`: https://github.com/jeandeaual/tts-deckconverter/blob/9d8637b162f684ec8f131d1d3f099d385191017b/tts/builder.go#L250